libqtile/backend/wayland/window.py

```diff
@@ -732,6 +732,12 @@ def static(
         self.defunct = True
         if self.group:
             self.group.remove(self)
+
+        conf_x = x
```

Comment:
just to help our future selves
```suggestion
        # Keep track of user-specified geometry to support X11
        # configure requests only if these are `None` here
        conf_x = x
```